doc/classes/TabContainer.xml

```diff
@@ -160,38 +165,39 @@
 			<argument index="0" name="visible" type="bool">
 			</argument>
 			<description>
-				If [code]true[/code] all the tabs will be visible.
+				If [code]false[/code], all tabs will be hidden completely (tab, content, and title). Defaults to [code]true[/code].
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="current_tab" type="int" setter="set_current_tab" getter="get_current_tab">
-			The current tab.
+			The current tab index. When set, this index's [class Control] node's [code]visible[/code] property is set to [code]true[/code] and all others are set to [code]false[/code].
 		</member>
 		<member name="tab_align" type="int" setter="set_tab_align" getter="get_tab_align" enum="TabContainer.TabAlign">
-			The alignment of all the tabs of the tab container. See the [code]ALIGN_*[/code] constants.
+			The alignment of all tabs in the tab container. See the [code]ALIGN_*[/code] constants for details.
 		</member>
 		<member name="tabs_visible" type="bool" setter="set_tabs_visible" getter="are_tabs_visible">
-			If [code]true[/code] all tabs that are children of the TabContainer will be visible.
+			If [code]true[/code] tabs are visible. If [code]false[/code], all tabs are hidden: the tabs and each one's content and title. Default value: [code]true[/code].
```

Comment:
remove the comma before "all tabs are hidden".
Remove the ":", the sentence could be:

    If [code]false[/code] all tabs' content and titles are hidden.

Reply:
fixed